gossip/src/contact_info.rs

```diff
@@ -234,6 +234,10 @@ impl ContactInfo {
     pub fn wallclock(&self) -> u64 {
         self.wallclock
     }
+    #[inline]
+    pub fn outset(&self) -> u64 {
+        self.outset
```

Comment:
outset is not meant to be used to compare nodes of different pubkey.

Reply:
we can ignore the outset and assume different identities can point to the same tvu address and just send shreds to both of them, i.e. remove the dedup logic from here.